### PR TITLE
Feature: Show current value in debugger when widget property has an error

### PR DESCRIPTION
**Pull request by akash-codemonk**
## Description
Current value is shown along with the error in the debugger 

## Type of change

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Cypress test

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:red_circle: Total coverage has decreased</summary>

    // Code coverage diff between base branch:release and head branch: feat/show-eval-value 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :red_circle: | total | 51.61 **(0)** | 32.43 **(-0.01)** | 29.82 **(0)** | 52.09 **(0)**
 :red_circle: | app/client/src/sagas/EvaluationsSaga.ts | 20.83 **(0)** | 1.79 **(-0.13)** | 10 **(0)** | 23.62 **(0)**</details>

### CODE REVIEW

File: app/client/src/sagas/EvaluationsSaga.ts [hunk 1]
```diff
@@ -114,6 +114,9 @@ const evalErrorHandler = (errors: EvalError[]) => {
           text: `The value at ${error.context?.source.propertyPath} is invalid`,
           message: error.message,
           source: error.context?.source,
+          state: {
+            ...error.context?.state,
+          },
```
Comment (rishabhsaxena) [inline app/client/src/sagas/EvaluationsSaga.ts:119]: Just curious, does it cause any issues if we don't create a new object here?

**Review by rishabhsaxena — DISMISSED** [review]

File: app/client/src/sagas/EvaluationsSaga.ts (hunk 1 above)
Comment (akash-codemonk) [reply app/client/src/sagas/EvaluationsSaga.ts:119]: Nope there will be no issues.

**Review by rishabhsaxena — APPROVED** [review]

**Pull request merged**